seed_data/challenges/computer-science.json

```diff
@@ -113,8 +130,12 @@
       "descriptionFr": [],
       "nameRu": "",
       "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
+      "nameEs": "Waypoint: Aprende sobre la Lógica Booleana",
```

Comment:
Little detail: this course isn't "learn about Boolean Logic" but "learn Boolean Logic", so the translation should say *"Aprende Lógica Booleana"* (which stresses more what the content is about).